hal/src/boron/hal_platform_config.h

```diff
@@ -28,6 +28,18 @@
 #define HAL_PLATFORM_RADIO_ANTENNA_EXTERNAL (1)
 #define HAL_PLATFORM_FLASH_MX25L3233F (1)
 
+#elif PLATFORM_ID == PLATFORM_ESOMX
+
+#define HAL_PLATFORM_POWER_MANAGEMENT (1)
+#define HAL_PLATFORM_PMIC_BQ24195 (1)
+#define HAL_PLATFORM_PMIC_BQ24195_FAULT_COUNT_THRESHOLD (4)
+#define HAL_PLATFORM_FUELGAUGE_MAX17043 (1)
+#define HAL_PLATFORM_PMIC_BQ24195_I2C (HAL_I2C_INTERFACE2)
+#define HAL_PLATFORM_FUELGAUGE_MAX17043_I2C (HAL_I2C_INTERFACE2)
+#define HAL_PLATFORM_RADIO_ANTENNA_INTERNAL (0)
+#define HAL_PLATFORM_RADIO_ANTENNA_EXTERNAL (1)
+#define HAL_PLATFORM_FLASH_MX25L3233F (1)
```

Comment:
If there is using 8M flash, need to change macro `#define HAL_PLATFORM_FLASH_MX25L3233F (1)` to `#define HAL_PLATFORM_FLASH_MX25R6435FZNIL0 (1)`